components/builder-jobsrv/src/server/worker_manager.rs

```diff
@@ -152,7 +152,6 @@ impl WorkerMgr {
                 try!(self.distribute_work());
             }
         }
-        Ok(())
```

Comment:
A lot of these warnings were as easy as just removing the code that I thought I originally needed for the compiler to pass ;)